src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -21,109 +26,258 @@
 	
 	@Autowired
 	private EvalCommentRepository commentDao;
+
+    @Autowired
+    private QuestionLogic questionLogic;
+
+    @Autowired
+    private JsonValidation validation;
+	
+	@PersistenceContext
+    private EntityManager entityManager;
 	
 
 //CREATE----------------------------
 	@Override
-	@Transactional
 	public Eval createEval(Eval eval) {
-		dao.save(eval);
+		
+		System.out.println(eval);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)